### PR TITLE
MLSS: General bugfixes + Add patch extension to inno_setup.iss

### PR DESCRIPTION
**Pull request by jamesbrq**
## What is this fixing or adding?
This fixes a GBA bios softlock due to the header of the game being edited.

## How was this tested?
The ROM was able to be verified in a fresh game and I could send checks out.

### CODE REVIEW

**Review by nicholassaylor — APPROVED** [review]
Tested old and new verification and changes are fixed like described, LGTM

**Comment by jamesbrq** [comment]
I accidentally used my main fork for this PR when I initially used it, so I just updated the PR title since 2 other small other changes we're made. Nicholas' review is now out of date on this PR.

File: inno_setup.iss [hunk 1]
```diff
@@ -169,6 +169,11 @@ Root: HKCR; Subkey: "{#MyAppName}pkmnepatch";                     ValueData: "Ar
 Root: HKCR; Subkey: "{#MyAppName}pkmnepatch\DefaultIcon";         ValueData: "{app}\ArchipelagoBizHawkClient.exe,0";                                          ValueType: string; ValueName: "";
 Root: HKCR; Subkey: "{#MyAppName}pkmnepatch\shell\open\command";  ValueData: """{app}\ArchipelagoBizHawkClient.exe"" ""%1""";                                 ValueType: string; ValueName: "";
 
+Root: HKCR; Subkey: ".apmlss";                                 ValueData: "{#MyAppName}cv64patch";                               Flags: uninsdeletevalue; ValueType: string; ValueName: "";
```
Comment (Silvris) [inline inno_setup.iss:172]:
```suggestion
Root: HKCR; Subkey: ".apmlss";                                 ValueData: "{#MyAppName}mlsspatch";                               Flags: uninsdeletevalue; ValueType: string; ValueName: "";
```

**Review by nicholassaylor — APPROVED** [review]
Additional changes also look good, renewing my review for approval

**Comment by black-sliver** [comment]
Can someone test the setup changes? Easiest is probably running it from the CI build ( https://github.com/ArchipelagoMW/Archipelago/actions/runs/9134849817 ) , which would also test frozen 3.8, which I think we also never test :sweat:

**Review by beauxq — APPROVED** [review]
- downloaded the Windows Python 3.8 build produced by CI
- installed it
- made the template yamls
- moved `Mario & Luigi Superstar Saga.yaml` to `Players`
- generated
- double-clicked the patch, and it asked me for my rom

I stopped there because I don't have the game, but it seems to be working to that point.

also skimmed over the code, and don't see any problems

**Review by NewSoupVi — APPROVED** [review]

**Pull request merged**